scripts/change-log-generator-utils.js

```diff
@@ -283,6 +285,32 @@ function getReleaseDate() {
     }).format(releaseDate);
 }
 
+/**
+ *
+ * Complete the heavy lifting to update the changelog by grabbing the
+ * new commits, grouping everything, and creating the text for editing.
+ * @param {string} remoteReleaseBranch
+ * @param {string} remotePreviousBranch
+ * @returns
+ */
+
+function updateChangeLog(remoteReleaseBranch, remotePreviousBranch) {
+  const parsedCommits = parseCommits(getCommits(remoteReleaseBranch, remotePreviousBranch));
+  if (parsedCommits.length > 0) {
+    const localReleaseBranch = remoteReleaseBranch.replace(constants.REMOTE_RELEASE_PREFIX_ONLY, '');
+    console.log(`\nChecking out ${localReleaseBranch}`);
+    const commitCommand = `git checkout ${localReleaseBranch}`;
+    shell.exec(commitCommand);
+    
+    const groupedMessages = getMessagesGroupedByPackage(parsedCommits, '');
+    const changeLog = getChangeLogText(remoteReleaseBranch, groupedMessages);
+    writeChangeLog(changeLog);
+  } else {
+    console.log(`No commits found, so we can skip this week's release. Carry on!`);
+    process.exit(0);
+  }
+}
+
 module.exports = {
-  getPreviousReleaseBranch, parseCommits, getMessagesGroupedByPackage, getChangeLogText, getCommits, writeChangeLog
+  updateChangeLog
```

Comment:
These were previously all being referenced in `create-release-branch.js` and I opted to simplify it all to a single function that needed exporting.